packages/@aws-cdk/aws-ec2/test/instance.test.ts

```diff
@@ -226,7 +247,7 @@ nodeunitShim({
 
       // THEN
       test.deepEqual(instance.node.metadata[0].type, cxschema.ArtifactMetadataEntryType.WARN);
-      test.deepEqual(instance.node.metadata[0].data, 'iops will be ignored without volumeType: EbsDeviceVolumeType.IO1');
+      test.deepEqual(instance.node.metadata[0].data, 'iops will be ignored without volumeType: EbsDeviceVolumeType.IO1 or EbsDeviceVolumeType.IO2');
```

Comment:
This warning is confusing. From the code it seems like setting volumeType to `EbsDeviceVolumeType.IO2` without setting iops is not allowed, but the warning implies the other way around? 

Same comment as below, if this test is not adding any value we shouldn't add it.

If these were added to satisfy the linter rule, note that I have added an exempt label so it will not block the PR

Reply:
Great! As the comment above, I'll remove this test too.